contracts/LoanManager.sol

```diff
@@ -72,12 +73,26 @@ contract LoanManager is Restricted, TokenReceiver {
         rates = _rates;
     }
 
+    /* @Deprecated: For compatibility with previous versions (without the margin parameter), e.g. old SB scripts */
     function addLoanProduct(uint32 term, uint32 discountRate, uint32 collateralRatio, uint minDisbursedAmount,
```

Comment:
Do we keep this temporary only? No client code uses it only local migration scripts